### ISSUE TITLE
1.3-Timeline Layout-Michael Esposito

### ISSUE THREAD

**Issue by Esponage** [TIY-GVL-FEE-2015-August/assignments#55]
# html-intro-4-timeline

## Description

This assignment serves to exercise layout and positioning with CSS.

## Objectives

- exercise control of CSS display types, positioning, animations
- reinforce concepts of layout and `display` types

### Learning Objectives

After completing this assignment, you should be able to create a layout in HTML and CSS from scratch.

### Performance Objectives

After completing this assignment, you should be able to effectively:

- create complex, multi-column layouts, with a variety of different methods; combining multiple approaches to achieve truly creative interface potential

### Deliverables

* A repo containing at least:
    - `index.html`
    - stylesheets
    - `images` folder
        - with any images you used with your design

### Tasks

* **Get Started**
    * [x] Open a new Issue in the class repo and add these checkboxes to your Issue.
    * [x] Create a new repository named `1.3-timeline-layout`.
    * [x] Create and commit an `index.html` file
    * [x] Push your work into Github

* [x]  **Read Learn Layout** if you haven't already

* **Write Some Code**
    * [x] Checkpoint: Initial commit!
    * [x] Checkpoint: Basic page structure (HTML only)!
    * [x] Checkpoint: First line of CSS committed and pushed!
    * [x] Checkpoint: Two column layout achieved.
    * [x] Checkpoint: Colors are exactly right.
    * [x] Checkpoint: Icons in place.
    * [x] Checkpoint: Circles and lines finished.
    * [x] Checkpoint: Hover states and transitions are done. Fancy.
    * [x] Mission Complete

## Normal Mode

Recreate the following layout in HTML and CSS. Try to get it as close to pixel
perfect as possible. You may slice out the images manually, or use alternative
icons.

Use the `:hover` pseudoclass and the `transition` property to make the different
shades of blue change on hover.

![](./timeline.png)

## Hard Mode

Recreate the mobile layout using media queries.

![](./timeline-mobile.png)

## Additional Resources

- http://learnlayout.com/
- http://learn.shayhowe.com/
- List of HTML tags: https://developer.mozilla.org/en-US/docs/Web/HTML/Element
- List of HTML attributes: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
- All the CSS properties! https://developer.mozilla.org/en-US/docs/Web/CSS/Reference

**Comment by Esponage** [comment]
I pushed this to github-pages but then went in to correct an error where it was breaking over 1400px.  I tried to sync it again, but the github-pages is still showing the old code.  The master is correct.

https://github.com/Esponage/1.3-Timeline-Layout-v2

Comfort Level: 3.5
Time To Complete: 9 hours- though I did restart after 3 hours of attempting hard mode and went to normal.
I'm pretty happy with this one.  It's not perfect by far, but so much better than yesterdays.  My issues were getting the line from the icon to the next icon solid with no breaks.  Also had trouble moving the header bottom border up closer to the text.  Last issue was the position of the date and time, wanted them to be closer together but no matter how much I messed with them it just wasn't happening.
